packages/eslint-config-core-ts/__tests__/explicit-member-accessibility.test.ts

```diff
@@ -0,0 +1,162 @@
+import { suite } from 'uvu';
+import { getTester } from '@infinumjs/test-utils';
+
+import eslintConfig from '../index';
+
+const rule = '@typescript-eslint/explicit-member-accessibility';
+const { validate } = getTester({
+	filePath: __filename,
+	eslintConfig: eslintConfig as any,
+	rule,
+});
+
+const test = suite(rule);
+
+test('should allow classes with explicitly defined member accessibility (except on public constructors)', () =>
+	validate(`
+	class Animal {
+		private constructor(public breed, name) {
+			// Parameter property and constructor
+			this.animalName = name;
+		}
+		private animalName: string; // Property
+		public get name(): string {
+			// get accessor
+			return this.animalName;
+		}
+		public set name(value: string) {
+			// set accessor
+			this.animalName = value;
+		}
+		public walk() {
+			// method
+		}
+	}
+	`));
+
+test('should allow classes with explicitly defined non-public constructors ', () =>
+	validate(`
```

Comment:
This test it identical to the previous one, can you maybe emphasize what are you trying to test by removing code that is not needed?